### PR TITLE
update metadata for connectors

### PR DESCRIPTION
**Pull request by mojavelinux**
- add missing exchange URLs
- add missing release notes xrefs
- add or fix support levels
- fix inconsistent titles
- fix inconsistent descriptions

### CODE REVIEW

**Comment by mojavelinux** [comment]
These changes provide the necessary information for the migration script to accurately migrate the connectors to the new structure. cc: @sleslie23

File: modules/ROOT/pages/aggregator/aggregators-module.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#
 
 Aggregators Module v1.0
 
-Aggregators module contains different types of aggregators capable of storing values and releasing them all together based on the configuration specified. An aggregator is a component used to collect those values until a condition is met, then execute a processor’s chain with the list of aggregated elements as the payload.
+The Aggregators Module contains different types of aggregators capable of storing values and releasing them all together based on the configuration specified. An aggregator is a component used to collect those values until a condition is met, then execute a processor’s chain with the list of aggregated elements as the payload.
```
Comment (valkyrie69) [inline modules/ROOT/pages/aggregator/aggregators-module.adoc]:
```suggestion
Aggregators Module contains different types of aggregators capable of storing values and releasing them all together based on the configuration specified. An aggregator is a component used to collect those values until a condition is met, then execute a processor’s chain with the list of aggregated elements as the payload.
```

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc [hunk 2]
```diff
@@ -11,7 +11,8 @@ Amazon S3 Connector v5.5.0
 
 Guide: xref:connectors::amazon/amazon-s3-connector.adoc[Amazon S3 Connector]
 
-Release Notes: xref:release-notes::connector/amazon-s3-connector-release-notes-mule-4.adoc[Amazon S3 Connector Release Notes]
+Release Notes: xref:release-notes::connector/amazon-s3-connector-release-notes-mule-4.adoc[Amazon S3 Connector Release Notes] +
+Exchange: https://anypoint.mulesoft.com/exchange/com.mulesoft.connectors/mule-amazon-s3-connector[Amazon S3 connector in Anypoint Exchange]
```
Comment (valkyrie69) [inline modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: Exchange link was intentionally removed. We don't want it at the top of the page.

File: modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector.adoc [hunk 3]
```diff
@@ -9,6 +9,8 @@ The Anypoint MQ connector provides publish-subscribe messaging to Mule apps usin
 
 Anypoint MQ supports a wide range of messaging use cases, such as enterprise messaging patterns, asynchronous communications between microservices, and IoT. Anypoint MQ also has a REST API for publishing or consuming messages using frameworks such as node.js, Java, Go, and DevOps bash scripts.
 
+Release Notes: xref:release-notes::connector/anypoint-mq-connector-release-notes-mule-4.adoc[Anypoint MQ Connector Release Notes] +
+Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/anypoint-mq-connector/[Anypoint MQ Connector 3.x — Mule 4]
```
Comment (valkyrie69) [inline modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector.adoc]: Should be in See Also not the top of the page

Comment (valkyrie69) [inline modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector.adoc]: Should be in See Also not the top of the page

File: modules/ROOT/pages/as2/as2-connector-reference.adoc [hunk 4]
```diff
@@ -9,7 +9,8 @@ AS2 Connector Version 5.0.0
 
 The main class of an extension is the entry point from which configurations, connection providers, operations, and sources are declared.
 
-Release Notes: xref:release-notes::connector/as2-connector-release-notes-mule-4.adoc[AS2 ConnectorRelease Notes]
+Release Notes: xref:release-notes::connector/as2-connector-release-notes-mule-4.adoc[AS2 Connector Release Notes] +
+Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-as2-connector/[AS2 Connector]
```
Comment (valkyrie69) [inline modules/ROOT/pages/as2/as2-connector-reference.adoc]: Move to See Also

File: modules/ROOT/pages/as2/as2-connectorMule4.adoc [hunk 5]
```diff
@@ -17,8 +17,8 @@ The AS2 Connector enables trading partners to transmit and receive AS2 messages
 
 * Receive asynchronous MDN for the message sent
 
-https://docs.mulesoft.com/release-notes/connector/as2-connector-release-notes-mule-4[Release Notes]
-https://www.mulesoft.com/exchange/?search=AS2&type=connector[Anypoint Exchange]
+Release Notes: xref:release-notes::connector/as2-connector-release-notes-mule-4.adoc[AS2 Connector Release Notes] +
+Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-as2-connector/[AS2 Connector]
```
Comment (valkyrie69) [inline modules/ROOT/pages/as2/as2-connectorMule4.adoc]: Move to See Also

File: modules/ROOT/pages/compression/compression-module.adoc [hunk 6]
```diff
@@ -7,7 +7,7 @@ Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#
 
 Compression Module v2.0
 
-Anypoint Compression module supports and provides access to the most-used
+The Compression Module supports and provides access to the most-used
```
Comment (valkyrie69) [inline modules/ROOT/pages/compression/compression-module.adoc]: No "the" on any of the connectors. It is Compression Module

File: modules/ROOT/pages/edifact/edifact-edi-connector.adoc [hunk 7]
```diff
@@ -5,9 +5,13 @@ endif::[]
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
 
+EDIFACT EDI Connector v2.0
+
 The EDIFACT EDI connector lets you convert EDIFACT messages to and from
 DataWeave-compatible representations using lists and maps. This connector supports EDIFACT versions D.93A, D.95B, D.96A, D.97A, D.98B, D.99A, D.99B, D.00A, D.01B, D.02A, D.03A with supplied schema definitions for all standard messages.
 
+Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-edifact-extension/[EDIFACT EDI Connector]
```
Comment (valkyrie69) [inline modules/ROOT/pages/edifact/edifact-edi-connector.adoc]: All of these Exchange links are being moved to the See Also section at the bottom of the page. They were intentionally removed from the top of the file.

File: modules/ROOT/nav.adoc [hunk 8]
```diff
@@ -226,8 +226,8 @@
 * xref:netsuite/netsuite-openair-connector.adoc[NetSuite OpenAir Connector]
  ** xref:netsuite/netsuite-openair-connector-reference.adoc[NetSuite OpenAir Connector Reference]
  ** xref:netsuite/netsuite-studio-configure.adoc[Configure the NetSuite Connector in Studio]
-* xref:oauth/oauth-documentation.adoc[OAuth Module Reference]
-* xref:oauth/oauth2-provider-documentation-reference.adoc[OAuth2 Provider Module Reference]
+* xref:oauth/oauth-documentation.adoc[OAuth Module]
```
Comment (valkyrie69) [inline modules/ROOT/nav.adoc]: Both of these should have "Reference" in the title. They are references.

File: modules/ROOT/pages/aggregator/aggregators-module.adoc (hunk 1 above)
Comment (valkyrie69) [inline modules/ROOT/pages/aggregator/aggregators-module.adoc]: Aggregators Module. No "The"

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc (hunk 2 above)
Comment (valkyrie69) [inline modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: Exchange link should be in See Also

File: modules/ROOT/nav.adoc [hunk 9]
```diff
@@ -226,8 +226,8 @@
...
+* xref:oauth/oauth2-provider-documentation-reference.adoc[OAuth2 Provider Module]
```
Comment (valkyrie69) [inline modules/ROOT/nav.adoc]: "Reference" should be in. These are references docs.

**Review by valkyrie69 — CHANGES_REQUESTED** [review]

**Comment by mojavelinux** [comment]
@valkyrie69 I believe I have made all the changes you requested. The update also fixes numerous broken xrefs and image refs, which were validated by the migration script. If you need me to make any other changes, don't hesitate to request them.

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**